framework/src/postprocessors/AreaPostprocessor.C

```diff
@@ -18,6 +18,9 @@ AreaPostprocessor::validParams()
 
   params.addClassDescription("Computes the \"area\" or dimension - 1 \"volume\" of a given "
                              "boundary or boundaries in your mesh.");
+  // Other user objects may depend on this value on INITIAL
+  // preaux runs before most user objects, and side UOs run before all UOs
```

Comment:
This comment should be clarified. Elemental user objects *execute* before side user objects on a per-element basis, but side user objects are joined and finalized before element user objects

Reply:
better now?
this is actually:
GeneralUO needs BC computeRC needs SideUO(area)